common/Makefile.am

```diff
@@ -71,6 +71,6 @@ libcommon_la_SOURCES = \
   $(PIXMAN_SOURCES)
 
 libcommon_la_LIBADD = \
-  -lpthread \
+  -lpthread -lrt \
```

Comment:
I don't think this is needed - it seems to build OK without it. Am I missing a compile option?

Reply:
I'm fine with that. Thanks for the clarification.

The reason I raised it was there's probably a better way to add this with autotools. I'll investigate further when we've got more dependencies to play with.